dat/factions/spawn/civilian.lua

```diff
@@ -4,15 +4,27 @@ require("dat/factions/spawn/common.lua")
 -- @brief Spawns a single small ship.
 function spawn_patrol ()
     local pilots = {}
-    local civships = {{"Civilian Schroedinger", 8},
-                      {"Civilian Llama", 8},
-                      {"Civilian Gawain", 8},
-                      {"Civilian Hyena", 13}
-                     }
-    
-    local select = rnd.rnd(1, #civships)
+    local r = rnd.rnd()
+
+    if r < 0.5 then
```

Comment:
This should probably be based on presence. I don't think very unsafe (pirates, etc.) systems should have too many advertisers. Should only somewhat exist in systems that are very populated.